README.md

```
@@ -63,86 +58,137 @@ pip install -e ".[test]"
 
-# Get metadata from SDRF
-metadata = SdrfUtil.get_metadata_dict_from_sdrf(sdrf_path)
+# List PSM parquet files
+parquet_files = list(ParquetPathHandler.iter_parquet_dir('/path/to/project/'))
 ```
 
-## Testing
+## Project Structure
 
-Run the test suite:
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)